pkg/coderefs/coderefs.go

```diff
@@ -158,6 +160,19 @@ func Scan() {
 	}
 }
 
+// Sorts grep results by Path and then line number
+func sortGrepResults(lines grepResultLines) {
+	sort.Slice(lines, func(i, j int) bool {
```

Comment:
Since we already have the `grepResultLines` type, we can implement [`sort.Interface`](https://golang.org/pkg/sort/#Interface) on `grepResultsLines` and then call `sort.Sort(linesToSort)`, which would replace the need for a separate function for sorting results